src/raygun.rum.js

```diff
@@ -734,6 +748,28 @@ var raygunRumFactory = function (window, $, Raygun) {
                 }
             }
         }
+        
+        function postSuccessCallback() {
+            self.postAttempts = 0;
+        }
+
+        function postErrorCallback(response, url, payload) {
+            self.postAttempts ++;
```

Comment:
extraneous space

Reply:
Cheers - I'll make the formatting more consistent